code/src/Components/Radiobuttons.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+/* Array for plants */
+const favoritePlants = ["Monstera", "Cactus", "Begonia", "Oxalis", "Aloe Vera", "Fiddle Leaf Fig", "Chinese Money Plant", "Philodendron", "Umbrella tree"];
```

Comment:
I created a practice for Tjejer Kodar about plants and I had to google what "Palettblad" is called in English because that is my favourite ;)